README.md

```diff
@@ -370,6 +371,30 @@ therefore, would look like the following:
       size: 3432
 ```
 
+Trailer response headers are useful for transmitting additional fields after the message body, providing dynamically generated metadata such as integrity checks or post-processing status. Proxy Verifier supports sending and verifying trailer headers, as demonstrated below:
```

Comment:
Most of this will go in the `Traffic Replay Specification`. Here we'll probably simply say, 

> In addition to HTTP/2 trailer header replay discussed above, Proxy Verifier also supports trailer header verification, as demonstrated below: